core/domain/user_jobs_one_off_test.py

```diff
@@ -1570,3 +1570,43 @@ def test_standard_operation(self):
             'removed explorations 0, 1, 2\', 1]' %
             self.user_id]
         self.assertEqual(sorted(actual_output), sorted(expected_output))
+
+
+class FixUserSettingCreatedOnOneOffJobTests(test_utils.GenericTestBase):
+
+    EXP_ID = 'exp_id_1'
+    ALBERT_EMAIL = 'albert@example.com'
+    ALBERT_NAME = 'albert'
+
+    def setUp(self):
+        super(FixUserSettingCreatedOnOneOffJobTests, self).setUp()
+
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+
+        self.save_new_valid_exploration(
+            self.EXP_ID, self.albert_id, end_state_name='End')
+
+        exp_services.update_exploration(
+            self.albert_id, self.EXP_ID, [exp_domain.ExplorationChange({
+                'cmd': 'edit_exploration_property',
+                'property_name': 'objective',
+                'new_value': 'the objective'
+            })], 'Test edit')
+
+    def test_user_gets_processed(self):
```

Comment:
Please verify that the `UserSettingsModel`s have the `created_on` set to correct value after the migration.